db-developments/sql/_hny_union.sql

```diff
@@ -0,0 +1,49 @@
+/*
+DESCRIPTION:
+    Union all HPD source data (HNY and historic)
+
+*/
+
+-- HPD units data
+ALTER TABLE hpd_hny_units_by_building ADD COLUMN ogc_fid_text text;
+UPDATE hpd_hny_units_by_building SET ogc_fid_text = 'hny'||'-'||CAST(ogc_fid as text);
+ALTER TABLE hpd_hny_units_by_building DROP COLUMN ogc_fid;
+ALTER TABLE hpd_hny_units_by_building RENAME COLUMN ogc_fid_text TO ogc_fid;
+
+ALTER TABLE _INIT_HPD_historical_units_by_building ADD COLUMN ogc_fid_text text;
+UPDATE _INIT_HPD_historical_units_by_building SET ogc_fid_text = 'historical'||'-'||CAST(ogc_fid as text);
+ALTER TABLE _INIT_HPD_historical_units_by_building DROP COLUMN ogc_fid;
+ALTER TABLE _INIT_HPD_historical_units_by_building RENAME COLUMN ogc_fid_text TO ogc_fid;
+
+DROP TABLE IF EXISTS HPD_units_by_building;
+SELECT 
+    *
+INTO 
+    HPD_units_by_building
+FROM (
+    SELECT * FROM hpd_hny_units_by_building
+    UNION
+    SELECT * FROM _INIT_HPD_historical_units_by_building
+) all_hpd_units;
```

Comment:
Just a bit nitpicky, but this can be a bit more succinct with just something like
```sql
CREATE TABLE HPD_units_by_building AS
SELECT * FROM hpd_hny_units_by_building
UNION
SELECT * FROM _INIT_HPD_historical_units_by_building
```